documentation/src/main/asciidoc/manual-index.asciidoc

```diff
@@ -398,6 +401,114 @@ long executionId = BatchRuntime
 ----
 ====
 
+==== Job Parameters
+
+.Job Parameters in JSR 352 Integration
+|===
+|Parameter Name |Builder Method |Requirement |Default value |Description
+
+|`rootEntities`
+|`forEntity(Class<?>)`, `forEntities(Class<?>, Class<?>...)`
+|Required
+|-
+|The root entity types to index in this job execution.
```

Comment:
I used the word "root" because it was used in several places in Hibernate Search. I'll remove it in a fix-up commit.

Should I update the related Javadoc and rename the input arguments of methods `forEntity(Class<?>)`, `forEntities(Class<?>, Class<?>...)`? (not the name of methods, but the name of arguments) Or maybe it's better to do it separately, in another JIRA ticket?